pkgs/development/python-modules/blinkstick/default.nix

```diff
@@ -1,35 +1,25 @@
-{ lib
-, buildPythonPackage
-, fetchPypi
-, pyusb
-}:
+{ lib, buildPythonPackage, fetchFromGitHub, pyusb }:
 
 buildPythonPackage rec {
   pname = "blinkstick";
   version = "1.2.0";
```

Comment:
might need to change this to `"unstable-YYYY-MM-dd"` format since there is no proper version

Reply:
many thanks @eclairevoyant for the input.